tasks/set-version.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * A task that updates version numbers in project files in accordance with
+ * build config.
+ *
+ * @param {Object} grunt The grunt helper object.
+ * @returns {void}
+ */
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"set-version",
+		"Configures a new version number.",
```

Comment:
If I understand this function correctly, it *sets* a new version number and doesn't configure it.